buildci.sh

```diff
@@ -35,6 +35,7 @@ if [ ! -z "$CHANGED_DIR" ] && [ -e "bundles/$CHANGED_DIR" ]; then
     echo "New addon pull request: Building $CHANGED_DIR"
     echo "MAVEN_OPTS='-Xms1g -Xmx2g -Dorg.slf4j.simpleLogger.log.org.openhab.tools.analysis.report.ReportUtility=DEBUG -Dorg.slf4j.simpleLogger.defaultLogLevel=WARN'" > ~/.mavenrc
     cd "bundles/$CHANGED_DIR"
+    mvn clean install -B 2>&1 | 
```

Comment:
This looks wrong to me

Reply:
I hope this it the reason that the script is failing to fail to build, because it looked so promising...